lnwallet/channel.go

```diff
@@ -8200,7 +8201,7 @@ func NewAnchorResolution(chanState *channeldb.OpenChannel,
 	return &AnchorResolution{
 		CommitAnchor:         *outPoint,
 		AnchorSignDescriptor: *signDesc,
-		CommitWeight:         weight,
+		CommitWeight:         lntypes.WeightUnit(weight),
```

Comment:
surprised that `weight` isn't already a `WeightUnit`. Looking upwards it seems like maybe this comes from `btcsuite`?

Reply:
Yeah not in `btcsuite` yet, IMO we should also propagate these types all the way up, as we have standard functions that return the weight/visze of a transaction.